bazel/scripts/setup_bazel_bash_completion.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+########################################################
+# Configures Bazel Bash-completion for the invoking user
+########################################################
+
+set -euo pipefail
+
+function log() {
+  echo "$@" >&2
```

Comment:
Why would you want to write standard logging to the error "channel"?

Reply:
That's the default behaviour for Linux commandline programs.